mbed_tools_ci_scripts/utils/filesystem_helpers.py

```diff
@@ -119,14 +119,17 @@ def __exit__(self, exc: Any, value: Any, tb: Any) -> None:
         """Context manager exit point."""
         self.cleanup()
 
+    def _cleanup_using_lib_facility(self) -> None:
+        try:
+            self._tmp_dir_context_manager.cleanup()
+        except FileNotFoundError as e:
+            logger.warning(
+                f"Failed cleaning up {self.path}. Reason: {str(e)}")
+
     def cleanup(self) -> None:
         """Deletes the temporary directory."""
-        if not (platform.system() == 'Windows'):
-            try:
-                self._tmp_dir_context_manager.cleanup()
-            except FileNotFoundError as e:
-                logger.warning(
-                    f"Failed cleaning up {self.path}. Reason: {str(e)}")
-        else:
+        if platform.system() == 'Windows':
             if self.path.exists():

```

Comment:
Pretty sure using `ignore_errors` makes this conditional obsolete.

Reply:
it is to avoid getting things in the logs for no real reasons

Reply:
What would be an example?